src/python/pants/backend/cc/dependency_inference/rules.py

```diff
@@ -65,10 +65,44 @@ class CCFilesMapping:
     mapping: FrozenDict[str, Address]
     ambiguous_files: FrozenDict[str, tuple[Address, ...]]
     mapping_not_stripped: FrozenDict[str, Address]
+    mapping_relative_to_include_dirs: FrozenDict[str, Address]
+
+
+def _map_relative_to_include_directories(
+    cc_file_mapping: dict[str, Address], include_directories: Iterable[str]
```

Comment:
Maybe `dict` -> `Mapping`?

Reply:
Ah, yep - bad habit on my part - I keep forgetting about Mappable, while always enforcing the use of Iterable 🤦🏽